### PR TITLE
Randomized Engines

### PR DESCRIPTION
**Pull request by monster860**
image

screenshot included so you know I tested it

:cl: monster860
add: The engine will now randomly be either a singulo or an SM
/:cl:

### CODE REVIEW

**Comment by Kmc2000** [comment]
cool

**Comment by ThatLing** [comment]
very cool cj

File: _maps/yogstation/RandomRuins/StationRuins/BoxStation/engine_singulo.dmm [hunk 1]
```diff
@@ -0,0 +1,1707 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```
Comment (Amelia0010) [inline _maps/yogstation/RandomRuins/StationRuins/BoxStation/engine_singulo.dmm:1]: could you be a doll and put external airlocks at the bottom corners of the singulo pen, like on prebase?

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

File: _maps/yogstation/RandomRuins/StationRuins/BoxStation/engine_singulo.dmm (hunk 1 above)
Comment (monster860) [reply _maps/yogstation/RandomRuins/StationRuins/BoxStation/engine_singulo.dmm:1]: done

**Review by Amelia0010 — CHANGES_REQUESTED** [review]
Decal dirs were changed in https://github.com/tgstation/tgstation/pull/42261, mirrored in https://github.com/yogstation13/Yogstation-TG/pull/4141. To fix your decals, update your branch to latest master then run the fixcorners.txt script by doing the following after opening command line in the tools/mapmerge2 folder:

python -m update_paths --map [path-for-the-map-you're-updating] [path-for-the-fixcorners-script,-will-end-in-tools/mapmerge2/map_scripts/fixcorners.txt

**Comment by monster860** [comment]
there done

**Review by Amelia0010 — APPROVED** [review]

**Pull request merged**